src/main/java/MentorObject.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
+
+public class MentorObject {
+  private String name;
+  private String description;
```

Comment:
We should discuss how we want to structure our data before we commit to this, but this definitely works for now. Take a look at what we have in the data doc we made yesterday and feel free to add your thoughts.

Reply:
For example, I'm wondering if the `name` property should be part of a more general User class, as both mentors and mentees will have names.

Reply:
For sure! Let's touch base during sunk.